main.js

```diff
@@ -0,0 +1,21 @@
+const tabitems=document.querySelectorAll('.tab-item');
+const tabcontentitems=document.querySelectorAll('.tab-content-item');
```

Comment:
tabitems and tabcontent items are declared const   which is a good practice since they are not reassigned